.chloggen/spanmetricsconnector-disabled.yaml

```
@@ -0,0 +1,21 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: spanmetricsconnector
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Histograms will not have exemplars by default"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [23872]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: "Previously spanmetricsconnector would attach every single span as an exemplar to the histogram. |
+  Exemplars are now disabled by default. To enable them set `exemplars.enabled=true`"
```

You need to start with a `|` instead of separating lines with a `|`, like this:
```suggestion
subtext: | 
  Previously spanmetricsconnector would attach every single span as an exemplar to the histogram.
  Exemplars are now disabled by default. To enable them set `exemplars::enabled=true`
```